openapi/external.yaml

```diff
@@ -35,7 +36,129 @@ paths:
           description: See Others
         "400":
           description: Validation Failed
-        "404":
-          description: Not Found
+  /api/v2/validate-profile-email:
+    get:
+      operationId: getTokenInfo
+      description: |
+        Validates the token and get email relative to a token.
+      parameters:
+        - in: query
```

Comment:
transport this value in header `x-pagopa-email-validation-token`

Reply:
done in https://github.com/pagopa/io-functions-public/pull/76/commits/317e91956f0411a4d23dc3131b3cae2bbcb7caff